action.yml

```diff
@@ -21,7 +21,7 @@ runs:
   using: 'node12'
   main: 'dist/restore/index.js'
   post: 'dist/save/index.js'
-  post-if: 'success()'
+  post-if: success() && env.CACHE_SKIP_SAVE != 'true'
```

Comment:
My only suggestion would be to have a more specific environment variable here.

Something like `GH_ACTIONS_CACHE_SKIP_SAVE`